spec/spec.yaml

```diff
@@ -3071,3 +3074,42 @@ paths:
           description: error
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
+
+  /v5/exception-notifications/:
+    post:
+      operationId: createExceptionNotification
```

Comment:
just noticed: In the operationId we use the verb `add` in cases like `addCluster`. Maybe align this here? (Might be that we are using `create` in some cases, too)

Reply:
yeah we have `create` and `add` used without any specific rule. I'll change this to `add` here though while we can.